src/pkgdev/scripts/pkgdev_mask.py

```diff
@@ -51,12 +51,27 @@
         Add a reference to a bug in the mask comment. May be specified multiple
         times to reference multiple bugs.
     """)
+mask_opts.add_argument(
+    '--email', dest='email', action='store_true',
+    help='spawn e-mail composer with prepared email for sending to mailing lists',
```

Comment:
Use `e-mail` or `email` consistently here and later in docs, personally I prefer `email` to match the option name.